### DISCUSSION TITLE
Open source dashboard date

### DISCUSSION THREAD

**Discussion by florian-lefebvre** [supabase/supabase#1608] [Questions] [answered]
Hi, do you have a date for the release of the open source dashboard?
Related: https://github.com/supabase/supabase/discussions/1299

**Comment by awalias** [comment, marked as answer]
hey @florian-lefebvre hard to know an exact launch date but @kiwicopple is building it in public so can track the progress there, try the dev version, or even contribute if you fancy:

https://www.youtube.com/watch?v=LHYrqBb4q9I

**Reply by florian-lefebvre** [reply]
Thanks @awalias !
